packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -67,77 +67,73 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     let datapoint: number | undefined = 0;
     return (
       <div className={this._classNames.root} onMouseLeave={this._handleChartMouseLeave}>
-        {data!.map((points: IChartProps, index: number) => {
-          if (points.chartData && points.chartData![0] && points.chartData![0].horizontalBarChartdata!.x) {
-            datapoint = points.chartData![0].horizontalBarChartdata!.x;
-          } else {
-            datapoint = 0;
-          }
-          points.chartData![1] = {
-            legend: '',
-            horizontalBarChartdata: {
-              x: points.chartData![0].horizontalBarChartdata!.y - datapoint!,
-              y: points.chartData![0].horizontalBarChartdata!.y,
-            },
-            color: palette.neutralLight,
-          };
+        <FocusZone direction={FocusZoneDirection.vertical} handleTabKey={FocusZoneTabbableElements.all}>
```

Comment:
In line 100 in previous implementation direction is set as horizontal. Here you have set it as vertical. Is this expected

Reply:
did undo the changes because horizontal navigation is required for stacked bars